modules/core/src/main/java/org/apache/ignite/plugin/extensions/communication/TimeLoggableResponse.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.plugin.extensions.communication;
+
+/**
+ * Common interface for responses.
+ */
+public interface TimeLoggableResponse extends Message {
```

Comment:
Please keep consistency in methods naming. I mean: rename req* methods to request*methods, i.e requestSentTimestamp OR rename response* methods to resp*.